tests/acceptance/common.py

```diff
@@ -222,13 +222,16 @@ def determine_active_passive_part(bitbake_variables, conn):
         )
 
 
-def get_ssh_common_args():
-    return "-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
+def get_ssh_common_args(conn):
+    args = "-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
+    if "key_filename" in conn.connect_kwargs.keys():
```

Comment:
for backwards compatibility maybe lets do `conn=None` and here add an `if conn:` ? (just a suggestion)

Reply:
I think I'll leave it as it is.

I would say that the normal case should be having a key, and that Yocto builds are the weird ones (it is created specially for testing that allows root acess with no password). So IOW, I expect this to be the regular path for other future images from mender-convert.